app/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.application'
```

Comment:
change this to apply plugin:'com.android.library'  so it doesn't complile to an application